extension/src/ALObject/ALElementTypes.ts

```diff
@@ -481,6 +481,20 @@ export class MultiLanguageObject extends ALElement {
     this.parent = parent;
   }
 
+  public set name(newName: string) {
+    const nameWithQuotesPattern = /^"(?<name>.+)"$/i;
+    const nameMatchResult = nameWithQuotesPattern.exec(newName);
```

Comment:
I thought that we usually declared a new instance of `RegExp` for these. But a quick search shows that we use both. Do we prefer one over the other or do we simply not care? :)
```suggestion
    const nameWithQuotesPattern = new RegExp(/^"(?<name>.+)"$/, "i";
    const nameMatchResult = nameWithQuotesPattern.exec(newName);
```

Reply:
I do not know the difference between
`const nameWithQuotesPattern = /^"(?<name>.+)"$/i`
or
`const nameWithQuotesPattern = new RegExp(/^"(?<name>.+)"$/, "i");`
or
`const nameWithQuotesPattern = new RegExp(/^"(?<name>.+)"$/i);`

All results in a RegExp object, right?

I assume that the different variants in use in this project has different copy&paste origins... ;)

Reply:
I guess the type is inferred with the first example. The end result is the same so the only argument here that I could find would be consistency but that's a discussion for another day :sweat_smile:

Reply:
Yeah, I agree that consistency is good. But I'm not ready for refactoring tonight... 😜